auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.kt

```diff
@@ -46,11 +46,10 @@ import java.security.PublicKey
 
 /**
  * API client for Auth0 Authentication API.
- * <pre>
- * `Auth0 auth0 = new Auth0("your_client_id", "your_domain");
- * AuthenticationAPIClient client = new AuthenticationAPIClient(auth0);
-` *
-</pre> *
+ * ```
+ * val auth0 = Auth0("your_client_id", "your_domain")
```

Comment:
we probably should be consistent with the use of these "placeholders" and the casing

```suggestion
 * val auth0 = Auth0("YOUR_CLIENT_ID", "YOUR_DOMAIN")
```

Reply:
New commit to address.